### PR TITLE
change config of env variables of travis

### PR DESCRIPTION
**Pull request by agnes512**
#### Related issues

- connect to https://github.com/strongloop/loopback-connector-mysql/issues/378

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,5 +3,20 @@ node_js:
...
+    - TEST_MYSQL_USER=test
+    - TEST_MYSQL_PASSWORD=test
+    - MYSQL_HOST=localhost
+    - MYSQL_PORT=3306
+    - MYSQL_USER=test
+    - MYSQL_PASSWORD=test
 before_script:
-  - "mysql -e 'create database myapp_test;'"
+  - mysql -u root -e "CREATE USER 'test'@'localhost' IDENTIFIED BY 'test';"
+  - mysql -e "use mysql; update user set authentication_string=PASSWORD('test') where User='test'; update user set plugin='mysql_native_password';FLUSH PRIVILEGES;"
```
Comment (dhmlau) [inline .travis.yml]: I wonder where the database `mysql` is created.  
From this line https://github.com/strongloop/loopback-connector-mysql/blob/master/test/init.js#L17, it seems like we are using `myapp_test` by default?

Comment (jannyHou) [reply .travis.yml]: looking at https://docs.travis-ci.com/user/database-setup/#mysql, looks like `mysql` is a default schema for user/system management.

`myapp_test` is the schema created for tests.
cc @dhmlau just my understanding :-p

File: .travis.yml [hunk 2]
```diff
@@ -3,5 +3,21 @@ node_js:
   - "6"
   - "8"
   - "10"
+services:
+  - mysql
+env:
+  global:
+    # - TEST_MYSQL_HOST=localhost
```
Comment (nabdelgadir) [inline .travis.yml]: Why are these lines commented?

File: .travis.yml (hunk 1 above)
Comment (dhmlau) [reply .travis.yml]: thx

**Comment by agnes512** [comment]
@slnode test please

**Comment by b-admike** [comment]
@slnode test please

**Comment by agnes512** [comment]
@slnode test please

**Comment by bajtos** [comment]
Can we use the MySQL server provided by Travis CI, instead of installing one via `apt-get`? See https://docs.travis-ci.com/user/database-setup/#mysql I would hope that the built-in MySQL instance will be faster too bootstrap.

Other than that, the changes LGTM.

**Comment by b-admike** [comment]
> Can we use the MySQL server provided by Travis CI, instead of installing one via `apt-get`? See https://docs.travis-ci.com/user/database-setup/#mysql I would hope that the built-in MySQL instance will be faster too bootstrap.
> 
> Other than that, the changes LGTM.

Good point. We wanted to pin the MySQL version to `5.7` and thus the extra `addons` section as suggested in https://docs.travis-ci.com/user/database-setup/#mysql-57. But I also see that it's the default for the `Xenial` image, so we can just use that for the build instead 👍.

**Pull request merged**